www/front_src/src/Resources/Filter/api/endpoint.ts

```diff
@@ -3,10 +3,11 @@ import {
   BuildListingEndpointParameters,
 } from '@centreon/ui';
 
-import { monitoringEndpoint } from '../../api/endpoint';
+import { baseEndpoint, monitoringEndpoint } from '../../api/endpoint';
 
 const hostgroupsEndpoint = `${monitoringEndpoint}/hostgroups`;
 const serviceGroupsEndpoint = `${monitoringEndpoint}/servicegroups`;
+const monitoringServersEndpoint = `${baseEndpoint}/configuration/monitoring-servers`;
```

Comment:
Why the Endpoint named "configuration/...." and the variable is "monitoring" ?

Reply:
cc @adr-mo

Reply:
@jdelpierre That is because right now it is currently implemented on configuration. No endpoint exists for now on the monitoring side.

Reply:
Ok, but I think it's a shame not to have consistency on the use for new things.
Simple question to understand: 
It's hard to make a dedicated endpoint? who can for example use classes or endpoint configuration if you want to behind?

Reply:
Replaced by a monitoring endpoint